### PR TITLE
style(frontend): Always order CSS properties in one way

### PR DESCRIPTION
**Pull request by Twixes**
## Changes

A follow-up to #18627, all this PR adds is a enforcing a single shared way of ordering CSS properties, improving readability of styles.
I initially considered alphabetic ordering, but that has one big drawback – closely related properties such as `height` and `width` naturally end up in completely different places.
Instead went for this sorting config: https://github.com/stormwarning/stylelint-config-recess-order. It's a popular style and basically what I've been doing without a formatter for a long time. Take a glance at the diff to see what this looks like.

### CODE REVIEW

**Review by benjackwhite — COMMENTED** [review]
Not against the change but please please let's not merge until I have the big refactor in

**Comment by daibhin** [comment]
Same as Ben, would love to wait until the buttons PR is merged too

**Review by daibhin — APPROVED** [review]
Will probably need to be rebased but totally happy for this to go through now. Don't think there is any major CSS branches ongoing right now

**Pull request merged**
